axelrod/tests/strategies/test_player.py

```diff
@@ -270,7 +307,7 @@ def attribute_equality_test(self, player, clone):
     def test_clone(self):
         # Test that the cloned player produces identical play
         player1 = self.player()
-        if str(player1) in ["Darwin", "Human"]:
+        if str(player1)[:6] in ["Darwin", "Human:"]:
```

Comment:
Because Human now get is str in the form 'Human: human_name, symbol1, symbol2' so I can't test the exact name so I did a split and I needed the same number of letters for Darwin and Human, hence the name. But I could replace that line by :
```python
if player1.name in ["Darwin", "Human"]:
```
it is more explicit.
But that got me to identify a little bug :
```python
In [5]: h= axl.Human('coco')

In [6]: str(h)
Out[6]: 'coco: coco, C, D'
```
So the question is : Do we want `str(h)` in that case to return 'Human: coco, C, D' or 'coco: C, D' ? I'll correct my code with the solution choosen